core/src/main/java/hudson/util/RingBufferLogHandler.java

```diff
@@ -35,25 +35,19 @@
  */
 public class RingBufferLogHandler extends Handler {
 
-    private static final int DEFAULT_RING_BUFFER_SIZE = Integer.getInteger(RingBufferLogHandler.class.getName() + ".defaultSize", 256);
-
     private int start = 0;
     private final LogRecord[] records;
     private volatile int size = 0;
 
-    /**
-     * This constructor is deprecated. It can't access system properties with {@link jenkins.util.SystemProperties}
-     * as it's not legal to use it on remoting agents.
-     */
-    @Deprecated
-    public RingBufferLogHandler() {
```

Comment:
Even though it's deprecated, it's good to keep it just in case it's being used by any plugin.